test/Project.toml

```diff
@@ -0,0 +1,79 @@
+[deps]
```

Comment:
What happened to this file? Why so many dependencies?

Reply:
file is deleted